packages/crayons-core/src/components/data-table/custom-cells/collection/custom-cell-collection.tsx

```diff
@@ -0,0 +1,72 @@
+import { Component, h, Prop, State, Host, Element } from '@stencil/core';
+
+@Component({
+  tag: 'fw-custom-cell-collection',
+  styleUrl: 'custom-cell-collection.scss',
+  shadow: true,
+})
+export class CustomCellCollection {
+  @Element() el: HTMLElement;
+
+  @Prop() collection = [];
+
+  @Prop() show = 2;
+
+  @State() open = false;
+
+  showMoreButton = null;
+
+  toggleCollection() {
+    this.open = !this.open;
+  }
+
+  onFocus() {
+    if (this.showMoreButton) {
+      this.showMoreButton.focus();
+    } else {
+      this.el.parentElement.setAttribute('tabindex', '0');
+      this.el.parentElement.focus();
+    }
+  }
+
+  render() {
+    return (
+      <Host onFocus={() => this.onFocus()}>
+        <div>
+          {this.collection.length <= this.show ? (
+            <span>{this.collection.join(', ')}</span>
+          ) : (
+            <span>
+              <span>
+                {this.open
+                  ? this.collection.join(', ')
+                  : this.collection.slice(0, this.show).join(', ')}{' '}
```

Comment:
please remove the `{' '}` in the end.

Reply:
This is necessary for space between the collection text and the show-more/show-less text.